\chapter{Pathway comparison} \label{paths-comparison}

\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between Pa-AD-f and Pa-AD-m.}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Pa-m AND AD-Pa-f & Synaptic signaling & 13 & 2.4 e-13 \\
                    & Synapse & 15 & 1.49 e-12 \\
                    & Secretory vesicle & 12 & 1.97 e-10 \\
                    & Cell-cell signaling & 14 & 1.07 e-9 \\
                    & Neuron projection & 12 & 5.42 e-9 \\
AD-Pa-m &
  CRHBP &
  Related to signaling by GPCR, presynaptic function of kainate receptors &
   \\
 &
  PCP4 &
  Related to regulation of neuron differentiation &
   \\
 &
  MET &
 Oncogene. Related to RET signaling, apoptotic pathways in synovial fibroblasts and GPCR pathway &
   \\
 &
  OPALIN &
  Oligodendrocytic Myelin Paranodal And Inner Loop. It increases myelin genes expression during oligodendrocyte differentiation, and promotes oligodendrocyte terminal differentiation &
   \\
AD-Pa-f &
  Synapse &
  14 &
  1.67 e-7 \\
 &
  Neuron projection &
  13 &
  1.2 e-6 \\
 &
  Synaptic vesicle membrane &
  5 &
  1.49 e-6 \\
 &
  Poor speech &
  6 &
  3.12 e-6 \\
 &
  Abnormality of the palpebral fissures &
  9 &
  3.26 e-6 \\
 &
  Abnormality of the forehead &
  10 &
  4.67 e-6 \\
 &
  Muscle weakness &
  10 &
  4.67 e-6 \\
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between Pa-AD-f and Pa-AD-m.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Pa-m AND AD-Pa-f & Cell junction assembly                    & 9 & 3.08 e-11 \\
                    & Synaptic growth at neuromuscular junction & 3 & 5.01 e-8  \\
                    & Cytoskeleton organization                 & 9 & 7.43 e-7  \\
                    & Cell substrate junction                   & 6 & 7.87 e-7  \\
                    & Actin binding                             & 6 & 9.5 e-7   \\
                    & Postsynapse assembly                      & 3 & 1.86 e-6  \\
AD-Pa-m             & Upper joint dislocation                   & 3 & 1.5 e-6   \\
                    & Absent frontal sinuses                    & 2 & 1.92 e-6  \\
                    & Abnormality of frontal sinuses            & 2 & 6.22 e-6  \\
                    & Abnormality of bone mineral density       & 4 & 7.26 e-6  \\
AD-Pa-f             & Leber optic atrophy                       & 7 & 6.29 e-19 \\
                    & Retinal arterial tortuosity               & 7 & 6.29 e-19 \\
                    & Mitochondrial respiratory chain defects   & 7 & 6.03 e-17 \\
                    & Blurred vision                            & 7 & 6.3 e-15  \\
                    & Action tremor                             & 9 & 8.83 e-14 \\
                    & Polyneuropathy                            & 7 & 3.72 e-13 \\
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between Temp-AD-f and Temp-AD-m.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Temp-m AND AD-Temp-f & Synaptic signaling                       & 12 & 4.24 e-12 \\
                        & Synapse                                  & 14 & 1.67 e-11 \\
                        & Cell-cell signaling                      & 14 & 7.23 e-10 \\
                        & Neuron projection                        & 10 & 5.29 e-7  \\
                        & Behavior                                 & 7  & 1.77 e-6  \\
                        & Neuropeptide signaling pathway           & 4  & 3.54 e-6  \\
AD-Temp-m                              & Regulation of neuron projection development & 6                          & 6.76 e-9         \\
                        & Regulation of cell development           & 7  & 1.11 e-8  \\
                        & Regulation of neuron differentiation     & 6  & 9.77 e-7  \\
                        & Regulation of axogenesis                 & 3  & 1.23 e-5  \\
AD-Temp-f               & Inhibitory synapse assembly              & 4  & 8.72 e-9  \\
                        & Synapse                                  & 13 & 1.05 e-7  \\
                        & Response to histamine                    & 3  & 1.87 e-7  \\
                        & Neuron projection                        & 12 & 8.72 e-7  \\
                        & GABA gated chloride ion channel activity & 3  & 9.5 e-7   \\
                        & GABA receptor activity                   & 3  & 5.06 e-6  \\
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between Temp-AD-f and Temp-AD-m.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Temp-m AND -f & Cell junction assembly                        & 6  & 2.99 e-7  \\
                 & Chronic axonal neuropathy                     & 2  & 4.04 e-6  \\
                 & Abnormal peripheral nervous system morphology & 7  & 7.44 e-7  \\
                 & Actin binding                                 & 5  & 8.21 e-6  \\
                 & Skeletal muscle atrophy                       & 7  & 8.38 e-6  \\
AD-Temp-m        & Glomerular epithelial cell differentiation    & 2  & 3.15 e-1  \\
                 & Glomerular epithelium development             & 2  & 3.15 e-1  \\
AD-Temp-f        & Biological adhesion                           & 29 & 1.69 e-17 \\
                 & Neurogenisis                                  & 30 & 4.65 e-17 \\
                 & Central nervous system development            & 24 & 4.55 e-16 \\
                 & Gliogenesis                                   & 15 & 5.23 e-15 \\
                 & Neuron differentiation                        & 24 & 3.08 e-13 \\
                 & Head development                              & 18 & 4.94 e-12 \\
                 & Neuron development                            & 19 & 1.95 e-10 \\
                 & Glial cell differentiation                    & 10 & 6.89 e-10\\
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between Hip-AD-f and Hip-AD-m.}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.4cm}L{8.1cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-HIP-m AND AD-HIP-f &
  Synaptic signaling &
  9 &
  1.82 e-13 \\
 &
  Synapse &
  10 &
  5.01 e-13 \\
 &
  Neuron projection &
  9 &
  3.83 e-11 \\
 &
  Axon &
  7 &
  4.19 e-10 \\
 &
  Cell-cell signaling &
  9 &
  4.63 e-10 \\
 &
  Nervous system process &
  7 &
  1.2 e-7 \\
AD-HIP-f &
  NCALD &
  A neuronal calcium sensor of calcium-binding proteins. Related to neurotransmitter receptor binding and downstream transmission in postsynaptic cell &
   \\
 &
  ROCK1P1 &
  Rho Associated Coiled-Coil Containing Protein Kinase 1 Pseudogene 1 &
   \\
 &
  RNVU1-7 &
  Affiliated with the snRNA class. Related to RNA transport and mRNA splicing pathways &
   \\
AD-HIP-m &
  Syntaxin binding &
  6 &
  8.48 e-10 \\
 &
  Secretory vesicle &
  12 &
  5.49 e-9 \\
 &
  Synapse &
  13 &
  1.28 e-7 \\
 &
  Intrinsic component of synaptic vesicle membrane &
  4 &
  4.31 e-7 \\
 &
  Neuron projection &
  11 &
  1.16 e-6 \\
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between Hip-AD-f and Hip-AD-m.}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-HIP-m & Response to topologically   incorrect protein & 7                                                                     & 1.47 e-10 \\
         & Inclusion body assembly                       & 4                                                                     & 2.68 e-9  \\
         & Signal transduction in absence of ligand      & 5                                                                     & 2.76 e-9  \\
         & Protein folding chaperone                     & 4                                                                     & 4.42 e-9  \\
         & Regulation of inclusion body assembly         & 3                                                                     & 8.29 e-8  \\
AD-HIP-m AND AD-HIP-f &
  AEBP1 &
  Related to DNA-binding transcription factor activity and calmodulin binding &
   \\
 &
  CDR1 &
  Cerebellar Degeneration Related Protein 1. Autoantibodies directed against this protein have been found in some patients with paraneoplastic cerebellar degeneration &
   \\
 &
  RNU11 &
  snRNA. Related to mRNA 5'-splice site recognition. Associated with microcephalic osteodysplastic primordial dwarfism &
   \\
         & LOC284412                                     & An RNA gene affiliated with the lncRNA class                          &           \\
         & MTSS1L                                        & Related to plasma membrane organization                               &           \\
AD-HIP-f &
  PCDHGA12 &
  A neural cadherin-like cell adhesion protein, plays a critical role in the establishment and function of specific cell-cell connections in the brain &
   \\
         & MT1F                                          & Related to metal ion SLC transporters and copper homeostasis pathways &           \\ 
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between Front-AD-f and Front-AD-m.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Front-m & Biological adhesion                 & 71  & 8.94 e-38  \\
           & Cell activation                     & 67  & 3.8 e-34   \\
           & Regulation of immune system process & 70  & 4.29 e-34  \\
                   & Regulation of multicellular orgranismal process & 70                         & 8.88 e-31        \\
           & Regulation of cell differentiation  & 70  & 1.67 e-29  \\
           & Response to cytokine                & 55  & 1.17 e-27  \\
AD-Front-f & Synapse                             & 171 & 1.52 e-126 \\
           & Neuron projection                   & 151 & 6.46 e-102 \\
           & Synaptic signaling                  & 103 & 2.23 e-77  \\
           & Axon                                & 92  & 1.48 e-70  \\
           & Glutamatergic synapse               & 73  & 4.26 e-67  \\
           & Somatodendritic compartment         & 95  & 2.12 e-62  \\ 
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between Front-AD-f and Front-AD-m.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Front-m           & Synapse                                                   & 232 & 7.76 e-184 \\
                     & Neuron projection                                         & 197 & 1.42 e-138 \\
                     & Synaptic signaling                                        & 197 & 8.25 e-116 \\
                     & Postsynapse                                               & 133 & 1.15 e-112 \\
                     & Presynapse                                                & 108 & 2.06 e-90  \\
                     & Somatodendritic compartment                               & 128 & 3.58 e-89  \\
AD-Front-f           & Biological   adhesion                                     & 101 & 1.1 e-55   \\
                     & Cell motility                                           & 101 & 1.78 e-47  \\
                     & Anatomical structure formation involved in morphogensis & 82  & 9.26 e-45  \\
                     & Regulation of immune system process                     & 93  & 2.54 e-44  \\
                     & Circulatory system development                          & 81  & 5.83 e-44  \\
                     & Regulation of cell differentiation                      & 98  & 2.54 e-42  \\
\multicolumn{1}{l}{} & Cell activation                                         & 85  & 9.47 e-36  \\
\multicolumn{1}{l}{} & Response to biotic stimulus                             & 82  & 1.3 e-30   \\
\multicolumn{1}{l}{} & Inflammatory response                                   & 58  & 1.98 e-29  \\
\multicolumn{1}{l}{} & Cytokine mediated signaling pathway                     & 56  & 1.68 e-27  \\
\bottomrule
\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between Fus-AD-f and Fus-AD-m.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-FUS-m AND AD-FUS-f & Synapse                     & 222 & 1.76 e-136 \\
                      & Neuron projection           & 198 & 1.75 e-110 \\
                      & Synaptic signaling          & 151 & 2.28 e-104 \\
                      & Cell-cell signaling         & 200 & 1.58 e-89  \\
                      & Postsynapse                 & 117 & 2.23 e-75  \\
AD-FUS-m              & 9 Plus 2 motile cilium      & 3   & 6.48 e-5   \\
AD-FUS-f              & Synapse                     & 90  & 1.62 e-42  \\
                      & Synaptic signaling          & 65  & 2.13 e-37  \\
                      & Neuron projection           & 80  & 7.09 e-34  \\
                      & Cell-cell signaling         & 90  & 8.7 e-33   \\
                      & Ion transport               & 83  & 2.46 e-26  \\
                      & Somatodendritic compartment & 57  & 6.59 e-26  \\
                      & Neuron differentiation      & 70  & 2.01 e-25  \\
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between Fus-AD-f and Fus-AD-m.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-FUS-m AND AD-Fus-f & Cell activation                                            & 83  & 2.36 e-41 \\
                & Positive regulation of multicellular orgranismal process & 88  & 3.27 e-38 \\
                & Biological   adhesion                                      & 79  & 1.21 e-37 \\
                & Anatomical structure formation involved in morphogensis  & 67  & 6.3 e-33  \\
                & Defense response                                         & 77  & 5.07 e-30 \\
AD-FUS-m        & Response to zinc ion                                     & 4   & 8.86 e-7  \\
                & Detoxification of inorganic compund                      & 3   & 1.55 e-6  \\
                & Cellular response to zinc ion                            & 3   & 4.59 e-6  \\
                & Cellular esponse to copper ion                          & 3   & 7.4 e-6   \\
AD-FUS-f        & Defense response                                           & 102 & 2.89 e-42 \\
                & Response to cytokine                                     & 83  & 4.94 e-40 \\
                & Regulation of immune system process                      & 91  & 3.99 e-37 \\
                & Cell   activation                                          & 85  & 9.47 e-36 \\
                & Response to biotic stimulus                              & 82  & 1.3 e-30  \\
                & Inflammatory response                                    & 58  & 1.98 e-29 \\
                & Cytokine mediated signaling pathway                      & 56  & 1.68 e-27 \\
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between PCx-HD and PCx-PD.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
HD-PCx AND PD-PCx                      & G protein coupled receptor signaling pathway & 17                         & 8.1 e-13         \\
       & Intrinsic component of plasma membrane   & 18 & 2.64 e-12 \\
       & Synaptic signaling                         & 12 & 1.79 e-10 \\
       & Response to dopamine                       & 5  & 1.29 e-7  \\
HD-PCx & Plasma membrane region                   & 22 & 1.59 e-6  \\
       & Intrinsic component of plasma membrane   & 26 & 3.97 e-6  \\
       & Intrinsic component of synaptic membrane & 8  & 4.31 e-6  \\
       & Synapse                                    & 22 & 2.63 e-2  \\
PD-PCx & Neuron projection                        & 26 & 2.71 e-17 \\
       & Synapse                                    & 25 & 2.65 e-16 \\
       & Neuropeptide signaling pathway           & 10 & 5.84 e-14 \\
       & Axon                                       & 15 & 1.83 e-11 \\
       & Cell-cell signaling                      & 22 & 7.78 e-11 \\
\bottomrule
\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between PCx-HD and PCx-PD.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
HD-PCx AND PD-PCx & Immune effector process                       & 55 & 6.91 e-43 \\
                  & Defense response                              & 57 & 2.07 e-37 \\
                  & Cell activation                               & 52 & 2.59 e-36 \\
                  & Leukocyte mediated immunity                 & 42 & 3.5 e-34  \\
                  & Myeloid leukocyte activation                & 37 & 1.4 e-32  \\
HD-PCx            & Defense response                              & 92 & 5.01 e-37 \\
                  & Response to biotic stimulus                 & 78 & 8.1 e-30  \\
                  & Tube development                            & 65 & 1.2 e-28  \\
                  & Tube morphogenesis                          & 58 & 3.19 e-27 \\
                  & Extracellular matrix                        & 46 & 7.79 e-27 \\
PD-PCx            & Secretion                                     & 23 & 4.02 e-13 \\
                  & Cell activation involved in immune response & 16 & 3.63 e-12 \\
                  & Immune effector process                     & 20 & 3.89 e-12 \\
                  & Leukocyte proliferation                     & 12 & 4.39 e-12 \\
\bottomrule
\end{tabularx}
\end{table}


\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between Blood-HD and PCx-HD.}
\begin{tabularx}{11cm}{L{2cm}L{2.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
HD-Blood-m & Abnormality of abdominal organs          & 24 & 1.48 e-10 \\
           & Cellular macromolecule localization    & 31 & 7.94 e-10 \\
           & Chromosome organization                & 22 & 3.81 e-8  \\
           & Biological adhesion                    & 22 & 6.58 e-7  \\
           & Ribonucleotide binding                 & 26 & 2.76 e-7  \\
HD-Blood-f & Cytosolic ribosome                     & 3  & 8.33 e-2  \\
           & Ribosomal subunit                      & 3  & 1.36 e-1  \\
           & Ribosome                                 & 3  & 2.51 e-1  \\
HD-PCx     & Intrinsic component of plasma membrane & 44 & 5.26 e-14 \\
           & Synapse                                  & 36 & 3.98 e-12 \\
           & Synaptic signaling                     & 26 & 1.66 e-11 \\
           & Neuron projection                      & 30 & 2.14 e-8  \\
           & Neurotransmitter receptor activity     & 9  & 7.64 e-8  \\
           & Synaptic membrane                      & 15 & 8.57 e-8  \\
\bottomrule
\end{tabularx}
\end{table}



\begin{table}[!ht]
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between Blood-HD and PCx-HD.}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
HD-Blood-m AND HD-PCx &
  CXCL1 &
  Plays a role in inflammation. Aberrant expression of this protein is associated with the growth of certain tumors &\\
 &
  CPVL &
  Related to proteolysis involved in cellular protein catabolic processes &\\
 &
  IL18R1 &
  It encodes a cytokine receptor that belongs to the interleukin 1 receptor family &\\
 &
  STX11 &
  Encodes a syntaxin, which has been implicated in the fusion of intracellular transport vesicles &\\
 &
  ANG &
  Potent mediator of new blood vessel formation. Related to cell junction organization and in lateral sclerosis &\\
 &
  ESM1 &
  May have potent implications in   lung endothelial cell-leukocyte interactions & \\
HD-PCx &
  Defense response &
  146 &
  1.53 e-68 \\
 &
  Response to biotic stimulus &
  128 &
  8.9 e-59 \\
 &
  Cell activation &
  122 &
  6.59 e-58 \\
 &
  Immune effector process &
  115 &
  9.17 e-58 \\
 &
  Inflammatory response &
  89 &
  2.41 e-52 \\
 &
  Myeloid leukocyte activation &
  82 &
  6.92 e-52 \\
HD-Blood-f &
  Specific granule lumen &
  3 &
  1.27 e-6 \\
HD-Blood-m &
  Abnormality of connective tissue &
  15 &
  2.59 e-7 \\
\ &
  Regulation of molecular function &
  16 &
  1.66 e-2 \\
 &
  Endoplasmic reticulum &
  14 &
  1.66 e-2 \\
\bottomrule
\end{tabularx}
\end{table}


\begin{table}[!ht]
\small
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between PCx-PD, PCx-HD and Front-AD.}
\begin{tabularx}{13cm}{L{2cm}L{4.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
HD-PCx AND PD-PCx                      & Protein coupled receptor signaling pathway & 13                         & 2.61 e-11        \\
                                 & Intrinsic component of plasma membrane       & 13  & 4.45 e-10  \\
                                 & Synaptic signaling                           & 9   & 7.02 e-9   \\
                                 & Response to endogenous stimulus              & 10  & 5.66 e-7   \\
                                 & Hormone mediated apoptotic signaling pathway & 2   & 6.57 e-6   \\
AD-Front-f AND HD-PCx            & Synapse                                        & 8   & 1.48 e-7   \\
                                 & Synaptic membrane                            & 5   & 1.13 e-6   \\
                                 & Postsynapse                                    & 6   & 5.19 e-7   \\
AD-Front-f AND PD-PCx            & Synapse                                        & 12  & 7.5 e-12   \\
                                 & Synaptic signaling                           & 10  & 1.28 e-11  \\
                                 & Axon                                           & 8   & 3.58 e-9   \\
                                 & Neuron projection                            & 10  & 4.2 e-9    \\
                                 & Terminal bouton                              & 3   & 5.82 e-6   \\
AD-Front-f AND PD-PCx AND HD-PCx & Synapse                                        & 6   & 4.09 e-1   \\
HD-PCx                           & No overlaps (lncRNA and siRNA)          &     &            \\
PD-PCx                           & Neuropeptide signaling pathway               & 8   & 9.27 e-12  \\
                                 & G protein coupled receptor signaling pathway & 17  & 5.9 e-11   \\
                                 & Nervous system process                       & 17  & 1.52 e-10  \\
                                 & Sensory perception                           & 12  & 6.62 e-8   \\
                                 & Nerve development                            & 4   & 9.19 e-6   \\
AD-Front-f                       & Synapse                                        & 145 & 1.86 e-105 \\
                                 & Neuron projection                            & 130 & 3.35 e-87  \\
                                 & Postsynapse                                    & 87  & 2.66 e-70  \\
                                 & Synaptic membrane                            & 71  & 1.18 e-66  \\
                                 & Ion transport                                & 105 & 4.95 e-51  \\
                                 & Neuron differentiation                       & 91  & 1.82 e-45  \\
                                 & Neurogenesis                                   & 98  & 2.65 e-45  \\
                                 & Glutamatergic synapse                        & 65  & 2.1 e-60   \\
AD-Front-m                       & Biological   adhesion                          & 73  & 1.03 e-38  \\
                                 & Cell activation                              & 68  & 3.52 e-34  \\
                                 & Regulation of immune system process          & 71  & 4.56 e-34  \\
                                 & Regulation of cell differentiation           & 72  & 3.1 e-30   \\
                                 & Response to cytokine                         & 55  & 5.78 e-27  \\
                                 & Apoptotic process                            & 64  & 2 e-23     \\ 
\bottomrule
\end{tabularx}
\end{table}


\begin{table}[!ht]
\tiny
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between PCx-PD, PCx-HD and Front-AD.}
\begin{tabularx}{13cm}{L{2cm}L{4.5cm}L{3cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
HD-PCx AND PD-PCx     & Immune effector process                       & 44  & 4.18 e-36  \\
                      & Leukocyte mediated immunity                 & 34  & 3.1 e-29   \\
                      & Cell activation                             & 40  & 7.92 e-29  \\
                      & Defense response                            & 43  & 8.87 e-29  \\
                      & Inflammatory response                         & 29  & 3.8 e-24   \\
                      & Secretion                                     & 36  & 6.51 e-23  \\
                      & Response to biotic stimulus                 & 40  & 2.42 e-27  \\
AD-Front-f AND HD-PCx & Epithelium development                      & 19  & 1.22 e-15  \\
                      & Regulation of cell differentiation          & 21  & 6.47 e-15  \\
                      & Biological adhesion                         & 18  & 1.31 e-13  \\
                      & Inflammatory response                         & 13  & 1.75 e-11  \\
                      & Defense response                            & 18  & 3.93 e-12  \\
AD-Front-f AND PD-PCx & Cell activation                               & 11  & 2 e-11     \\
                      & Cell activation involved in immune response & 9   & 2.61 e-11  \\
                      & Regulation of immune system process         & 10  & 1.6 e-9    \\
                      & Leukocyte proliferation                     & 6   & 7.65 e-9   \\
                      & Secretion                                     & 10  & 1.6 e-9    \\
AD-Front-f AND HD-PCx AND PD-PCx       & Defense response                                   & 14                         & 5.01 e-10        \\
                      & Apoptotic process                           & 14  & 1.71 e-9   \\
                      & Regulation of cell death                      & 11  & 2.04 e-9   \\
                      & Immune effector process                     & 11  & 2.03 e-8   \\
                      & Inflammatory response                         & 9   & 4.73 e-8   \\
HD-PCx                & Defense response                              & 74  & 1.98 e-27  \\
                      & Extracellular matrix                          & 41  & 2.83 e-24  \\
                      & Response to biotic stimulus                 & 63  & 2.66 e-22  \\
                      & Leukocyte mediated immunity                 & 46  & 3.02 e-21  \\
                      & Tube morphogenesis                          & 45  & 3.22 e-19  \\
                      & Inflammatory response                         & 40  & 5.59 e-18  \\
PD-PCx                & Transmembrane transport                     & 15  & 5.11 e-8   \\
                      & Response to abiotic stimulus                & 12  & 8.89 e-7   \\
                      & Secretion                                     & 13  & 2.4 e-6    \\
                      & Ion transport                               & 13  & 4.01 e-6   \\
                      & Sugar transmembrane transporter activity    & 3   & 1 e-5      \\
AD-Front-f            & Cell motility                                 & 75  & 1.67 e-36  \\
                      & Biological adhesion                         & 67  & 7.59 e-35  \\
                                       & Regulation of anatomical structure morphogenesis & 56                         & 1.05 e-29        \\
                      & Vascular development                        & 48  & 1.35 e-29  \\
                      & Tube morphogenesis                          & 46  & 5.2 e-25   \\
                      & Regulation of immune system process         & 58  & 1.24 e-24  \\
AD-Front-m            & Synapse                                       & 232 & 7.76 e-184 \\
                      & Neuron projection                           & 197 & 1.42 e-138 \\
                      & Cell-cell signaling                         & 172 & 2.27 e-89  \\
                      & Somatodendritic compartment                 & 128 & 3.58 e-89  \\
                      & Axon                                          & 115 & 4.86 e-89  \\
                      & Regulation of transport                     & 156 & 1.07 e-72  \\
                      & Neuron development                          & 127 & 1.25 e-72  \\
                      & Dendritic tree                              & 98  & 1.55 e-70  \\
\bottomrule
\end{tabularx}
\end{table}


\begin{table}[!ht]
\tiny
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between all male tissues with AD.}
\label{tab:broken1}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Pa-m AND AD-Fus-m &
  SPTSSB &
  Serine palmitoyltransferase catalyzes the rate-limiting step in sphingolipid biosynthesis &
   \\
 &
  PCP4 &
  Related to regulation of neuron differentiation &
   \\
 &
  MET &
 Oncogene. Related to RET signaling, apoptotic pathways in synovial fibroblasts and GPCR pathway &
   \\
 &
  ATP1B1 &
  An ATPase, which is an integral membrane protein responsible for establishing and maintaining the electrochemical gradients of Na and K ions across the plasma membrane &
   \\
 &
  GABRG2 &
  GABA receptor. GABA is the major inhibitory neurotransmitter in the mammalian brain &
   \\
AD-Fus-m AND AD-Hip-m &
  Neuron projection &
  9 &
  4.11 e-9 \\
 &
  Anion channel activity &
  3 &
  4.09 e-2 \\
 &
  GABA   gated channel activity &
  2 &
  4.09 e-2 \\
 &
  GABA   receptor complex &
  2 &
  4.55 e-2 \\
AD-Front-m AND AD-Pa-m &
  OPALIN &
  Central nervous system-specific myelin protein that increase myelin genes expression during oligodendrocyte differentiation and promotes oligodendrocyte terminal differentiation &
   \\
AD-Temp-m AND AD-Fus-m &
  Response to endogenous stimulus &
  7 &
  5.54 e-7 \\
 &
  Regulation of axogenesis &
  4 &
  2.54 e-6 \\
AD-Temp-m AND AD-Hip-m &
  PCDHGC5 &
  Neural cadherin-like cell adhesion protein plays a critical role in the establishment and function of specific cell-cell connections in the brain &
   \\
AD-Pa-m AND AD-Fus-m AND AD-Hip-m &
  RBM11 &
  Tissue-specific splicing factor with implication in the regulation of alternative splicing during neuron and germ cell differentiation &
   \\
 &
  GABRA1 &
  GABA is the major inhibitory neurotransmitter in the mammalian brain where it acts at GABA-A receptors &
   \\
 &
  STMN2 &
  Stathmin proteins function in microtubule dynamics and signal transduction. Plays a regulatory role in neuronal growth and it might be involved in osteogenesis &
   \\
 &
  CTXN3 &
  - &
   \\
AD-Temp-m AND AD-Pa-m AND AD-Fus-m &
  Secretory vesicle &
  4 &
  3.23 e-1 \\
 &
  Transport vesicle &
  3 &
  3.23 e-1 \\
AD-Temp-m AND AD-Pa-m AND AD-Hip-m &
  NME1 &
  Mutations in this gene have been identified in aggressive neuroblastomas &
   \\
AD-Temp-m AND AD-Fus-m AND AD-Hip-m &
  Syntaxin binding &
  2 &
  2.15 e-1 \\
AD-Temp-m AND AD-Pa-m AND AD-Fus-m AND AD-Hip-m &
  Synapse &
  12 &
  1.02 e-12 \\
 &
  Synaptic signaling &
  9 &
  1.09 e-10 \\
 &
  Behavior &
  7 &
  2.45 e-8 \\
 &
  Axon &
  7 &
  3.67 e-8 \\
 &
  Neurotransmitter secretion &
  5 &
  3.72 e-8 \\
 &
  Regulation of transport &
  8 &
  3.72 e-6 \\
\midrule
\multicolumn{4}{@{}r@{}}{continued on following page.}

\end{tabularx}
\end{table}  


\begin{table}[!ht]
\tiny
\centering
\caption{Table \ref{tab:broken1}, continued.}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Pa-m &
  CRHBP &
  Related to signaling by GPCR, presynaptic function of kainate receptors &
   \\
 &
  RNU2-2P &
  RNA, U2 Small Nuclear 2, Pseudogene &
   \\
 &
  KIAA0408 &
  Uncharacterized protein &
   \\
AD-Fus-m &
  Synapse &
  196 &
  4.34 e-115 \\
 &
  Neuron projection &
  172 &
  2.91 e-90 \\
 &
  Cell-cell signaling &
  178 &
  4.99 e-76 \\
 &
  Ion transport &
  153 &
  1.23 e-59 \\
 &
  Somatodendritic compartment &
  111 &
  2.05 e-58 \\
 &
  Axon &
  97 &
  3.48 e-57 \\
AD-Hip-m &
  Stem cell differentiation &
  3 &
  3.41 e-1 \\
AD-Temp-m &
  AMACR &
  Mutations in this gene may be associated with adult-onset sensorimotor neuropathy and adrenomyeloneuropathy due to defects in bile acid synthesis &
   \\
 &
  MAS1 &
  Related to this gene include G protein-coupled receptor activity and peptide hormone binding pathways &
   \\
 &
  LINCR-0003 &
  Uncharacterized LincR-0003 &
   \\
 &
  LINC01007 &
  Long Intergenic Non-Protein Coding RNA 1007 &
   \\
 &
  HAMP &
  This gene is involved in the maintenance of iron homeostasis &
   \\
AD-Front-m &
  Biological adhesion &
  73 &
  8.02 e-39 \\
 &
  Cell activation &
  68 &
  2.82 e-34 \\
 &
  Regulation of immune system process &
  71 &
  3.61 e-34 \\
 &
  Regulation of cell differentiation &
  71 &
  1.59 e-29 \\
 &
  Response to cytokine &
  55 &
  4.85 e-27 \\
 &
  Apoptotic process &
  64 &
  1.65 e-23 \\
 &
  Lymphocyte activation &
  41 &
  3.29 e-23 \\
\bottomrule

\end{tabularx}
\end{table}


\begin{table}[!ht]
\small
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between all male tissues with AD.}
\label{tab:broken2}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Pa-m   AND AD-Fus-m &
  SLCO4A1 &
  Related to transport of glucose and other compounds, vitamins, nucleotides and relates molecules &
   \\
AD-Pa-m AND AD-Hip-m &
  SHANK1 &
  Acts as scaffold protein that are required for the development and function of neuronal synapses &
   \\
 &
  FLNA &
  This protein interacts with integrins, transmembrane receptor complexes, and second messengers &
   \\
AD-Fus-m AND AD-Hip-m &
  Response to topologically incorrect protein &
  5 &
  6.78 e-11 \\
 &
  Inclusion body assembly &
  3 &
  6.6 e-9 \\
 &
  Signal transduction in absence of ligand &
  3 &
  2.19 e-7 \\
 &
  Unfolded protein binding &
  3 &
  8.64 e-7 \\
 &
  Protein folding &
  3 &
  6.42 e-6 \\
AD-Temp-m AND AD-Pa-m &
  Regulation of synapse structural plasticity &
  2 &
  1.61 e-6 \\
AD-Front-m AND AD-Pa-m &
  FGF17 &
  This protein is expressed during embryogenesis and in the adult cerebellum and cortex, and may be essential for vascular growth and normal brain development &
   \\
AD-Temp-m AND AD-Fus-m &
  KLF15 &
  Related to circadian gene expression and adipogenesis &
   \\
 &
  RAPGEF3 &
  Related to guanyl-nucleotide exchange factor activity and cAMP-dependent protein kinase regulator activity &
   \\
 &
  HIGD1B &
  The encoded protein is localized in the cell membrane and has been linked to tumorigenesis. &
   \\
 &
  TNS3 &
  May play a role in actin remodeling. Involved in the dissociation of the integrin-tensin-actin complex &
   \\
 &
  C7orf61 &
  Chromosome 7 Open Reading Frame 61 &
   \\
 &
  DDIT4 &
  Plays an important role in responses to cellular energy levels and cellular stress, including responses to hypoxia and DNA damage &
   \\
 &
  MT1F &
  Related to metal ion SLC transporters and copper homeostasis &
   \\
AD-Temp-m AND AD-Hip-m &
  MTSS1L &
  Related to plasma membrane organization &
   \\
AD-Pa-m AND AD-Fus-m AND AD-Hip-m &
  AEBP1 &
  Related to DNA-binding transcription factor activity and calmodulin binding &
   \\
 \midrule
\multicolumn{4}{@{}r@{}}{continued on following page.}

\end{tabularx}
\end{table}
   
   
\begin{table}[!ht]
\small
\centering
\caption{Table \ref{tab:broken2}, continued.}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Temp-m AND AD-Pa-m AND AD-Fus-m &
  LRP4 &
  Plays a role in the formation and the maintenance of the neuromuscular junction &
   \\
 &
  SMTN &
  Found exclusively in contractile smooth muscle cells. It associates with stress fibers and constitutes part of the cytoskeleton &
   \\
AD-Temp-m AND AD-Pa-m AND AD-Hip-m &
  No overlaps &
   &
   \\
AD-Temp-m AND AD-Fus-m AND AD-Hip-m &
  Regulation of DNA binding transcription factor activity &
  3 &
  2.15 e-2 \\
AD-Temp-m AND AD-Pa-m AND AD-Hip-m &
  HIF3A &
  Acts as a transcriptional regulator in adaptive response to low oxygen tension &
   \\
 &
  TNS1 &
  Localized to focal adhesions, regions of the plasma membrane where the cell attaches to the extracellular matrix &
   \\
 &
  HSPB1 &
  Through its molecular chaperone activity, it may regulate numerous biological processes including the phosphorylation and the axonal transport of neurofilament proteins &
   \\
 &
  MKNK2 &
  Plays important roles in the initiation of mRNA translation, oncogenic transformation and malignant cell proliferation &
   \\
AD-Pa-m &
  Regulation of cell differentiation &
  5 &
  1.8 e-1 \\
AD-Fus-m &
  Cell activation &
  88 &
  3.25 e-43 \\
 &
  Defense response &
  85 &
  7.34 e-34 \\
 &
  Inflammatory response &
 57 &
  9.88 e-32 \\
 &
  Tube development &
  62 &
  7.52 e-28 \\
 &
  Vascular development &
  52 &
  1.82 e-26 \\
AD-Hip-m &
  No overlaps &
   &
   \\
AD-Temp-m &
  No overlaps &
   &
   \\
AD-Front-m &
  Synapse &
 232 &
  4.88 e-184 \\
 &
  Neuron projection &
  197 &
  9.81 e-139 \\
 &
  Somatodendritic compartment &
  128 &
  2.86 e-89 \\
 &
  Regulation of transport &
  156 &
  8.22 e-73 \\
 &
  Ion transport &
  148 &
  5.52 e-71 \\
 &
  Neurogenisis &
  145 &
  5.12 e-69 \\
 &
  Dendritic tree &
  98 &
  1.31 e-70 \\
  \bottomrule
\end{tabularx}
\end{table}


\begin{table}[!ht]
\small
\centering
\caption{Comparison of over represented pathways according to the up-regulated differential expression results between all female tissues with AD.}
\label{tab:broken3}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Pa-f AND AD-Fus-f &
  Proton transporting V type ATPase V1 domain &
  2 &
  3.49 e-6 \\
 &
  Limb hypertonia &
  3 &
  3.51 e-6 \\
 &
  Ribonucleotide binding &
  6 &
  1.45 e-5 \\
AD-Pa-f AND AD-Hip-f &
  RNVU1-7 &
  Related pathways are RNA transport and mRNA Splicing &
   \\
AD-Temp-f AND AD-Pa-f &
  PRPS2 &
  Plays a role in the synthesis of purines and pyrimidines &
   \\
 &
  EIF4A2 &
  Cap recognition and is required for mRNA binding to ribosome &
   \\
 &
  ZNF702P &
  Related to gene expression pathway &
   \\
 &
  PLCXD2 &
  Related to lipid metabolic process and signal transduction &
   \\
AD-Temp-f AND AD-Fus-f &
  No overlaps &
   &
   \\
AD-Front-f AND AD-Fus-f &
  Synapse &
  130 &
  1.01 e-99 \\
 &
  Neuron projection &
  116 &
  4.93 e-82 \\
 &
  Somatodendritic compartment &
  79 &
  3.59 e-57 \\
 &
  Axon &
  68 &
  4.86 e-53 \\
 &
  Ion transport &
  94 &
  5.05 e-49 \\
 &
  Dendritic tree &
  62 &
  5.19 e-47 \\
  AD-Temp-f AND AD-Pa-f AND AD-Fus-f &
  Oxaloacetate metabolic process &
  2 &
  6.99 e-2 \\
AD-Front-f AND AD-Pa-f AND AD-Fus-f &
  Synaptic vesicle membrane &
  4 &
  9.67 e-7 \\
 &
  Secretory vesicle &
  7 &
  3.15 e-2 \\
AD-Temp-f AND AD-Fus-f AND AD-Hip-f &
  PVALB &
  The encoded protein might be involved in muscle relaxation &
   \\
 &
  SST &
  Somatostatin affects rates of neurotransmission in the central nervous system and proliferation of both normal and tumorigenic cells &
   \\
AD-Front-f AND AD-Temp-f AND AD-Pa-f &
  SV2C &
  Plays a role in the control of regulated secretion in neural and endocrine cells, enhancing selectively low-frequency neurotransmission &
   \\
 &
  KIAA0408 &
  Uncharacterized protein &
   \\
   AD-Front-f AND AD-Temp-f AND AD-Fus-f &
  Terminal bouton &
  2 &
  5.92 e-1 \\
 &
  Regulation of neuron differentiation &
  3 &
  6.62 e-1 \\
\midrule
\multicolumn{4}{@{}r@{}}{continued on following page.}

\end{tabularx}
\end{table}


\begin{table}[!ht]
\small
\centering
\caption{Table \ref{tab:broken3}, continued.}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value}\\
\midrule
AD-Front-f AND AD-Pa-f AND AD-Fus-f AND AD-Hip-f &
  SYT4 &
  Plays a role in dendrite formation by melanocytes &
   \\
AD-Front-f AND AD-Temp-f AND AD-Pa-f AND AD-Fus-f &
  Amino acid transport &
  4 &
  4.92 e-2 \\
 &
  Synapse &
  8 &
  4.92 e-2 \\
AD-Front-f AND AD-Temp-f AND AD-Pa-f AND AD-Fus-f AND AD-Hip-f &
  Synaptic signaling &
  7 &
  2.44 e-10 \\
 &
  Synapse &
  8 &
  2.55 e-10 \\
 &
  Neuron projection &
  7 &
  1.54 e-8 \\
 &
  Synaptic transmission GABAergic &
  3 &
  2.64 e-7 \\
AD-Pa-f &
  HIST1H2AE &
  Histones are basic nuclear proteins that are responsible for the nucleosome structure \\
 &
  CASQ1 &
  Plays an important role in triggering muscle contraction \\
 &
  HTR5A-AS1 &
  HTR5A Antisense RNA 1 &
   \\
 &
  PCDHGC5 &
  A neural cadherin-like cell adhesion protein which plays a role in the establishment and function of specific cell-cell connections in the brain &
   \\
 &
  RNU2-2P &
  RNA, U2 Small Nuclear 2, Pseudogene &
   \\
 &
  COPG2IT1 &
  lncRNA &
   \\
AD-Fus-f &
  Synapse &
  149 &
  3.8 e-66 \\
 &
  Synaptic signaling &
  114 &
  3.69 e-65 \\
 &
  Cell-cell signaling &
  161 &
  8.83 e-59 \\
 &
  Neuron projection &
  133 &
  1.28 e-52 \\
 &
  Regulation of transport &
  138 &
  4.42 e-41 \\
 &
  Behavior &
  76 &
  1.94 e-37 \\
 &
  Nervous system process &
  106 &
  1.81 e-30 \\
 &
  Secretion &
  110 &
  3.18 e-29 \\
AD-Hip-f &
  ROCK1P1 &
  Rho Associated Coiled-Coil Containing Protein Kinase 1 Pseudogene 1 &
   \\
AD-Temp-f &
  No overlaps &
   &
   \\
AD-Front-f &
  Synaptic membrane &
  10 &
  1.25 e-10 \\
 &
  Synapse &
  14 &
  2.9 e-9 \\
 &
  Neuron projection &
  13 &
  3.04 e-8 \\
 &
  Postsynaptic specialization membrane &
  5 &
  6.29 e-7 \\
 &
  Neurogenesis &
  12 &
  2.25 e-6 \\
 &
  Neuron spine &
  5 &
  4.62 e-6 \\
 &
  Cerebral cortex tangential migration &
  2 &
  2 e-2 \\
  \bottomrule
  
\end{tabularx}
\end{table}


\begin{table}[!ht]
\tiny
\centering
\caption{Comparison of over represented pathways according to the down-regulated differential expression results between all female tissues with AD.}
\label{tab:broken4}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.5cm}L{8cm}C{1.5cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value} \\
\midrule
AD-Pa-f AND AD-Fus-f &
  PDGFRB &
  This gene is essential for   normal development of the cardiovascular system and aids in rearrangement of the actin cytoskeleton &
   \\
 &
  VCAN &
  Plays a role in intercellular signaling and in connecting cells with the extracellular matrix. May take part in the regulation of cell motility, growth and differentiation &
   \\
AD-Temp-f AND AD-Pa-f &
  Respiratory system development &
  6 &
  1.58 e-4 \\
 &
  Central nervous system devolpment &
  9 &
  4.67 e-4 \\
 &
  Cell projection membrane &
  6 &
  8.11 e-4 \\
 &
  Regulation of neuron differentiation &
  7 &
  9.88 e-4 \\
 &
  Cell junction assembly &
  6 &
  9.88 e-4 \\
AD-Front-f AND AD-Pa-f &
  SASH1 &
  Potential tumor suppressor that may negatively regulate proliferation, apoptosis, and invasion of cancer cells &
   \\
 &
  MYO10 &
  Myosins are actin-based motor molecules with ATPase activity. Unconventional myosins serve in intracellular movements &
   \\
AD-Temp-f AND AD-Fus-f &
  COL27A1 &
  Plays a role during the calcification of cartilage and the transition of cartilage to bone &
   \\
 &
  MID1IP1 &
  Plays a role in the regulation of lipogenesis in liver, and is involved in stabilization of microtubules &
   \\
 &
  EFHD1 &
  Related to mitosis, synaptic transmission, and cytoskeletal rearrangement &
   \\
AD-Temp-f AND AD-Hip-f &
  MTSS1L &
  Related to plasma membrane organization &
   \\
 &
  PCDHGA12 &
  A neural cadherin-like celladhesion protein which plays a role in the establishment and function of specific cell-cell connections in the brain &
   \\
AD-Front-f AND AD-Fus-f &
  Anatomical structure formation involved in morphogensis &
  47 &
  1.33 e-31 \\
 &
  Regulation of immune system process &
  52 &
  9.39 e-31 \\
 &
  Cell activation &
  49 &
  9.03 e-30 \\
 &
  Response to cytokine &
  43 &
  5.81 e-27 \\
 &
  Vascular development &
  36 &
  1.07 e-25 \\
 &
  Regulation of cell differentiation &
  47 &
  1.19 e-22 \\
AD-Front-f AND AD-Temp-f &
  Nephrotic syndrome &
  3 &
  2.63 e-2 \\
 &
  Abnormality of urine homeostasis &
  4 &
  2.63 e-2 \\
 &
  Abnormal renal physiology &
  4 &
  2.63 e-2 \\
AD-Temp-f AND AD-Pa-f AND AD-Fus-f &
  KIF1C &
  Microtubule-dependent molecular motors that transport organelles within cells and move chromosomes during cell division &
   \\
 &
  HIPK2 &
  Inhibits cell growth and promotes apoptosis through the activation of p53 both at the transcription level and at the protein level &
   \\
 &
  MAP4K4 &
  Plays a role in the response to environmental stress and cytokines &
   \\
 &
  LPAR1 &
  Plays a role in the reorganization of the actin cytoskeleton, cell migration, differentiation and proliferation &
   \\
AD-Temp-f AND AD-Pa-f AND AD-Hip-f &
  LOC284412 &
  Uncharacterized LOC284412 &
   \\
 &
  RNU11 &
  snRNA. Related to mRNA 5'-splice site recognition. Related to microcephalic osteodysplastic primordial dwarfism &
   \\
 &
  CDR1 &
  Autoantibodies directed against this protein have been found in some patients with paraneoplastic cerebellar degeneration &
   \\
AD-Front-f AND AD-Pa-f AND AD-Fus-f &
  TNS2 &
  Tensin is a focal adhesion molecule that binds to actin filaments and participates in signaling pathways &
   \\
AD-Front-f AND AD-Fus-f AND AD-Hip-f &
  AEBP1 &
  Related to DNA-binding transcription factor activity and calmodulin binding &
   \\
\midrule
\multicolumn{4}{@{}r@{}}{continued on following page.}

\end{tabularx}
\end{table}



\begin{table}[!ht]
\small
\centering
\caption{Table \ref{tab:broken4}, continued.}
\begin{tabularx}{1.0\textwidth}{L{2cm}L{2.9cm}L{8cm}C{1.1cm}}
\toprule
\textbf{Intersection} & \textbf{Gene set} & \textbf{Overlapping genes} & \textbf{p-value}\\
\midrule
AD-Front-f AND AD-Temp-f AND AD-Pa-f &
  Rho protein signal transduction &
  3 &
  2.14 e-2 \\
 &
  Astrocyte projection &
  2 &
  3.3 e-2 \\
 &
  Glial cell projection &
  2 &
  5.62 e-2 \\
 &
  Oligodendrocyte development &
  2 &
  6.93 e-2 \\
AD-Front-f AND AD-Temp-f AND AD-Fus-f &
  CSF1 &
  A cytokine that controls the production, differentiation, and function of macrophages &
   \\
 &
  NUPR1 &
  Participates in regulation of cell-cycle, apoptosis, autophagy and DNA repair responses &
   \\
 &
  FGFRL1 &
  The extracellular portion of the protein interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, influencing mitogenesis and differentiation &
   \\
AD-Front-f AND AD-Temp-f AND AD-Pa-f AND AD-Fus-f &
  Tube morphogenesis &
  6 &
  2.62 e-7 \\
 &
  Blood vessel morphogenesis &
  5 &
  1.92 e-6 \\
 &
  Homeostatic process &
  6 &
  1.66 e-2 \\
AD-Front-f AND AD-Temp-f AND AD-Pa-f AND AD-Hip-f &
  MT1F &
  Related to metal ion SLC transporters and copper homeostasis &
   \\
AD-Pa-f &
  Leber optic atrophy &
  7 &
  6.45 e-23 \\
 &
  Retinal arterial tortuosity &
  7 &
  6.45 e-23 \\
 &
  Mitochondrial respiratory chain defects &
  7 &
  6.21 e-21 \\
 &
  Vascular tortuosity &
  7 &
  2.27 e-20 \\
AD-Fus-f &
  Cell activation &
  114 &
  8.78 e-47 \\
 &
  Defense response &
  124 &
  2.26 e-45 \\
 &
  Immune effector process &
  94 &
  2.91 e-36 \\
 &
  Inflammatory response &
  73 &
  3.67 e-35 \\
 &
  Leukocyte mediated immunity &
  70 &
  3.34 e-29 \\
AD-Temp-f &
  Biological adhesion &
  13 &
  1.53 e-7 \\
 &
  Neuron-neuron synapse &
  7 &
  1.05 e-6 \\
 &
  Central nervous system devolpment &
  10 &
  2.28 e-6 \\
 &
  Regulation of hydrolase activity &
  10 &
  3.13 e-2 \\
AD-Front-f &
  Biological adhesion &
  44 &
  1.04 e-23 \\
 &
  Regulation of cellular component movement &
  34 &
  2.34 e-18 \\
 &
  Regulation of cell differentiation &
  41 &
  9.37 e-17 \\
 &
  Response to cytokine &
  32 &
  1.05 e-15 \\
 &
  Apoptotic process &
  40 &
  1.25 e-15 \\
\bottomrule

\end{tabularx}
\end{table}